### ISSUE TITLE
Make site nav consistent in inactive, rollover, and active states

### ISSUE THREAD

**Issue by hauten** [LLNL/llnl.github.io#318]
For better site accessibility, a navigation option should change (e.g., text or background color) when the user mouses over it. This happens on most, but not all, of the site. We should be consistent. The behaviors in this issue are confined to the top-level site navigation.

No rollover or selection behavior:
- News, About, Explore, and their sub-pages - Nothing happens at rollover. When the page is selected, the background turns gray while the text remains black. Sub-pages are blue text with no rollover change.
- Don't forget about the footer links like the LLNL logo

Behavior exists but is inconsistent:
- RADIUSS - unlike the other top menu links, this one goes from light gray to dark gray, which isn't much contrast
- Catalog/category browses - light gray background turns to dark gray (text remains black) on rollover; while text on blue background when selected
- Twitter & GH icons in top menu - dark gray turns black; goes offsite so no selection behavior necessary
- Home - blue text turns dark gray on rollover; returns to blue when selected
- News, button at bottom of page that goes to Archive - blue on blue is bad (this is the biggest offender)

Ideally, every menu option should have the same text colors & background colors for (1) inactive, (2) rollover, and (3) active/selected. Ideally don't change the catalog/category browse behavior, so try to make the rest match that as best as possible without deviating too far from the overall Lab web template this site imitates.
